tests/mixed-source/ios/MixedSourceFramework/Logger/DoubleQuoteNamespacedLogger.m

```diff
@@ -0,0 +1,15 @@
+#import "DoubleQuoteNamespacedLogger.h"
+#import "MixedSourceFramework/MixedSourceFramework-Swift.h" // The `MixedSourceFramework-Swift.h` header allows
```

Comment:
I don't think there's anywhere else where we make namespaced quote imports work -- is this really needed?

Reply:
I am completely ambivalent about this.  

Since Objective-C source files are typically importing the generated Swift header from _within_ _the_ _same_ _framework_, it seems to me like the extra namespacing here is superfluous.  I added it because for whatever reason, the first framework I tried to build had authors who used namespaced imports for headers inside their own framework.  I don't know what advantage they may have gained by doing so.

So I'm happy to add a commit to this PR that removes the namespaced case from the test fixture and updates the `rules_ios` code to ensure that we don't allow namespaced header imports for the generated Swift header.